{
  "uid" : "671c21c531f0462e",
  "name" : "Check registration without choosing agreement",
  "fullName" : "tests.registration_page_test.TestRegistrationPage#test_registration_without_choosing_agreement",
  "historyId" : "372d09e576a89594cbbb625e370f331d",
  "time" : {
    "start" : 1728697313466,
    "stop" : 1728697313466,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5591810ec02a <unknown>\n#1 0x559180dd25e0 <unknown>\n#2 0x559180dca7eb <unknown>\n#3 0x559180dbaa09 <unknown>\n#4 0x559180dbc744 <unknown>\n#5 0x559180dbacbd <unknown>\n#6 0x559180dba52f <unknown>\n#7 0x559180dba422 <unknown>\n#8 0x559180db8493 <unknown>\n#9 0x559180db8ada <unknown>\n#10 0x559180dd4f41 <unknown>\n#11 0x559180e668c5 <unknown>\n#12 0x559180e46b22 <unknown>\n#13 0x559180e65d7d <unknown>\n#14 0x559180e468c3 <unknown>\n#15 0x559180e146b3 <unknown>\n#16 0x559180e1568e <unknown>\n#17 0x5591810b6a2b <unknown>\n#18 0x5591810ba9b1 <unknown>\n#19 0x5591810a3225 <unknown>\n#20 0x5591810bb532 <unknown>\n#21 0x55918108838f <unknown>\n#22 0x5591810daf28 <unknown>\n#23 0x5591810db0f3 <unknown>\n#24 0x5591810eae7c <unknown>\n#25 0x7ffac8894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"d4cc08b526d599aac8b950227869a8bf\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4df4f0ec30>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5591810db0f3 \\\\u003Cunknown>\\\\n#24 0x5591810eae7c \\\\u003Cunknown>\\\\n#25 0x7ffac8894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5591810ec02a <unknown>\nE       #1 0x559180dd25e0 <unknown>\nE       #2 0x559180dca7eb <unknown>\nE       #3 0x559180dbaa09 <unknown>\nE       #4 0x559180dbc744 <unknown>\nE       #5 0x559180dbacbd <unknown>\nE       #6 0x559180dba52f <unknown>\nE       #7 0x559180dba422 <unknown>\nE       #8 0x559180db8493 <unknown>\nE       #9 0x559180db8ada <unknown>\nE       #10 0x559180dd4f41 <unknown>\nE       #11 0x559180e668c5 <unknown>\nE       #12 0x559180e46b22 <unknown>\nE       #13 0x559180e65d7d <unknown>\nE       #14 0x559180e468c3 <unknown>\nE       #15 0x559180e146b3 <unknown>\nE       #16 0x559180e1568e <unknown>\nE       #17 0x5591810b6a2b <unknown>\nE       #18 0x5591810ba9b1 <unknown>\nE       #19 0x5591810a3225 <unknown>\nE       #20 0x5591810bb532 <unknown>\nE       #21 0x55918108838f <unknown>\nE       #22 0x5591810daf28 <unknown>\nE       #23 0x5591810db0f3 <unknown>\nE       #24 0x5591810eae7c <unknown>\nE       #25 0x7ffac8894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728697313466,
      "stop" : 1728697314231,
      "duration" : 765
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728697314232,
      "stop" : 1728697314654,
      "duration" : 422
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5591810ec02a <unknown>\n#1 0x559180dd25e0 <unknown>\n#2 0x559180dca7eb <unknown>\n#3 0x559180dbaa09 <unknown>\n#4 0x559180dbc744 <unknown>\n#5 0x559180dbacbd <unknown>\n#6 0x559180dba52f <unknown>\n#7 0x559180dba422 <unknown>\n#8 0x559180db8493 <unknown>\n#9 0x559180db8ada <unknown>\n#10 0x559180dd4f41 <unknown>\n#11 0x559180e668c5 <unknown>\n#12 0x559180e46b22 <unknown>\n#13 0x559180e65d7d <unknown>\n#14 0x559180e468c3 <unknown>\n#15 0x559180e146b3 <unknown>\n#16 0x559180e1568e <unknown>\n#17 0x5591810b6a2b <unknown>\n#18 0x5591810ba9b1 <unknown>\n#19 0x5591810a3225 <unknown>\n#20 0x5591810bb532 <unknown>\n#21 0x55918108838f <unknown>\n#22 0x5591810daf28 <unknown>\n#23 0x5591810db0f3 <unknown>\n#24 0x5591810eae7c <unknown>\n#25 0x7ffac8894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728697314232,
        "stop" : 1728697314654,
        "duration" : 422
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5591810ec02a <unknown>\n#1 0x559180dd25e0 <unknown>\n#2 0x559180dca7eb <unknown>\n#3 0x559180dbaa09 <unknown>\n#4 0x559180dbc744 <unknown>\n#5 0x559180dbacbd <unknown>\n#6 0x559180dba52f <unknown>\n#7 0x559180dba422 <unknown>\n#8 0x559180db8493 <unknown>\n#9 0x559180db8ada <unknown>\n#10 0x559180dd4f41 <unknown>\n#11 0x559180e668c5 <unknown>\n#12 0x559180e46b22 <unknown>\n#13 0x559180e65d7d <unknown>\n#14 0x559180e468c3 <unknown>\n#15 0x559180e146b3 <unknown>\n#16 0x559180e1568e <unknown>\n#17 0x5591810b6a2b <unknown>\n#18 0x5591810ba9b1 <unknown>\n#19 0x5591810a3225 <unknown>\n#20 0x5591810bb532 <unknown>\n#21 0x55918108838f <unknown>\n#22 0x5591810daf28 <unknown>\n#23 0x5591810db0f3 <unknown>\n#24 0x5591810eae7c <unknown>\n#25 0x7ffac8894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"d4cc08b526d599aac8b950227869a8bf\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728697167335,
      "stop" : 1728697167335,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728697314686,
      "stop" : 1728697314748,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Registration Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "registration_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestRegistrationPage"
  }, {
    "name" : "host",
    "value" : "fv-az570-490"
  }, {
    "name" : "thread",
    "value" : "2304-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.registration_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 58,
        "unknown" : 0,
        "total" : 60
      },
      "items" : [ {
        "uid" : "3e3e0442d5bba6bd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/3e3e0442d5bba6bd",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55cc36e7902a <unknown>\n#1 0x55cc36b5f5e0 <unknown>\n#2 0x55cc36b577eb <unknown>\n#3 0x55cc36b47a09 <unknown>\n#4 0x55cc36b49744 <unknown>\n#5 0x55cc36b47cbd <unknown>\n#6 0x55cc36b4752f <unknown>\n#7 0x55cc36b47422 <unknown>\n#8 0x55cc36b45493 <unknown>\n#9 0x55cc36b45ada <unknown>\n#10 0x55cc36b61f41 <unknown>\n#11 0x55cc36bf38c5 <unknown>\n#12 0x55cc36bd3b22 <unknown>\n#13 0x55cc36bf2d7d <unknown>\n#14 0x55cc36bd38c3 <unknown>\n#15 0x55cc36ba16b3 <unknown>\n#16 0x55cc36ba268e <unknown>\n#17 0x55cc36e43a2b <unknown>\n#18 0x55cc36e479b1 <unknown>\n#19 0x55cc36e30225 <unknown>\n#20 0x55cc36e48532 <unknown>\n#21 0x55cc36e1538f <unknown>\n#22 0x55cc36e67f28 <unknown>\n#23 0x55cc36e680f3 <unknown>\n#24 0x55cc36e77e7c <unknown>\n#25 0x7f4c7de94ac3 <unknown>",
        "time" : {
          "start" : 1728611004710,
          "stop" : 1728611004710,
          "duration" : 0
        }
      }, {
        "uid" : "f43061aa6ee9d061",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/f43061aa6ee9d061",
        "status" : "passed",
        "time" : {
          "start" : 1728524943433,
          "stop" : 1728524964180,
          "duration" : 20747
        }
      }, {
        "uid" : "ac115e71b9f19843",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/ac115e71b9f19843",
        "status" : "passed",
        "time" : {
          "start" : 1728438615227,
          "stop" : 1728438635981,
          "duration" : 20754
        }
      }, {
        "uid" : "a69f9186d548754a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/a69f9186d548754a",
        "status" : "passed",
        "time" : {
          "start" : 1728352197839,
          "stop" : 1728352218634,
          "duration" : 20795
        }
      }, {
        "uid" : "267d6ecc44046cac",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/267d6ecc44046cac",
        "status" : "passed",
        "time" : {
          "start" : 1728266609258,
          "stop" : 1728266630029,
          "duration" : 20771
        }
      }, {
        "uid" : "29eab13088543de",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/29eab13088543de",
        "status" : "passed",
        "time" : {
          "start" : 1728179742237,
          "stop" : 1728179762986,
          "duration" : 20749
        }
      }, {
        "uid" : "3f252095f59da75c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/3f252095f59da75c",
        "status" : "passed",
        "time" : {
          "start" : 1728092977109,
          "stop" : 1728092997855,
          "duration" : 20746
        }
      }, {
        "uid" : "bd9a1d6c45f4176d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/bd9a1d6c45f4176d",
        "status" : "passed",
        "time" : {
          "start" : 1728006680351,
          "stop" : 1728006701094,
          "duration" : 20743
        }
      }, {
        "uid" : "41dd8e0b96262020",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/41dd8e0b96262020",
        "status" : "passed",
        "time" : {
          "start" : 1727920323659,
          "stop" : 1727920344478,
          "duration" : 20819
        }
      }, {
        "uid" : "6cc791a9b5c30b9c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/6cc791a9b5c30b9c",
        "status" : "passed",
        "time" : {
          "start" : 1727833752708,
          "stop" : 1727833773476,
          "duration" : 20768
        }
      }, {
        "uid" : "8d9758d8dd034974",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/8d9758d8dd034974",
        "status" : "passed",
        "time" : {
          "start" : 1727747877302,
          "stop" : 1727747898088,
          "duration" : 20786
        }
      }, {
        "uid" : "c1eccc398fbc7406",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/c1eccc398fbc7406",
        "status" : "passed",
        "time" : {
          "start" : 1727661159534,
          "stop" : 1727661180294,
          "duration" : 20760
        }
      }, {
        "uid" : "4a4ee59ea4fe1467",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/4a4ee59ea4fe1467",
        "status" : "passed",
        "time" : {
          "start" : 1727574980460,
          "stop" : 1727575001210,
          "duration" : 20750
        }
      }, {
        "uid" : "2400fdd1d0ca92d2",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/2400fdd1d0ca92d2",
        "status" : "passed",
        "time" : {
          "start" : 1727488206795,
          "stop" : 1727488227536,
          "duration" : 20741
        }
      }, {
        "uid" : "170811b98effc773",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/170811b98effc773",
        "status" : "passed",
        "time" : {
          "start" : 1727401944842,
          "stop" : 1727401965599,
          "duration" : 20757
        }
      }, {
        "uid" : "8f4fd5067c2e262",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/8f4fd5067c2e262",
        "status" : "passed",
        "time" : {
          "start" : 1727315420495,
          "stop" : 1727315441226,
          "duration" : 20731
        }
      }, {
        "uid" : "cc7e0a53d7ad6c2a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/cc7e0a53d7ad6c2a",
        "status" : "passed",
        "time" : {
          "start" : 1727229073589,
          "stop" : 1727229094336,
          "duration" : 20747
        }
      }, {
        "uid" : "be70d133a3dc32d3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/be70d133a3dc32d3",
        "status" : "passed",
        "time" : {
          "start" : 1727142604706,
          "stop" : 1727142625493,
          "duration" : 20787
        }
      }, {
        "uid" : "b6d3a88dfb561363",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/b6d3a88dfb561363",
        "status" : "passed",
        "time" : {
          "start" : 1727056127078,
          "stop" : 1727056147790,
          "duration" : 20712
        }
      }, {
        "uid" : "929dc2e7f2fb26e3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/929dc2e7f2fb26e3",
        "status" : "passed",
        "time" : {
          "start" : 1726970055478,
          "stop" : 1726970076220,
          "duration" : 20742
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "671c21c531f0462e.json",
  "parameterValues" : [ ]
}